CMakeLists.txt

```diff
@@ -0,0 +1,209 @@
+### setup project ###
+# https://numpy.org/doc/stable/f2py/buildtools/skbuild.html
+cmake_minimum_required(VERSION 3.18)
+
+project(${SKBUILD_PROJECT_NAME}
+  VERSION ${SKBUILD_PROJECT_VERSION}
+  DESCRIPTION "Utilities for reading WRF output"
+  LANGUAGES C Fortran
+  )
+
+# Safety net
+if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
+  message(
+    FATAL_ERROR
+      "In-source builds not allowed. Please make a new directory (called a build directory) and run CMake from there.\n"
+  )
+endif()
+
+find_package(Python 3.8 REQUIRED COMPONENTS Development.Module NumPy)
```

Comment:
Right, the interpreter would be required to run f2py, so that should definitely be there.

I suppose most who would know to use CMake directly would also know to check pyproject.toml for version requirements, and this also lets `pip install --ignore-requires-python` work properly.